src/alert_rules/prometheus/prometheus_alerts.yaml

```diff
@@ -0,0 +1,89 @@
+"groups":
+- "name": "opensearch.alerts"
+  "rules":
+  - "alert": "OpenSearchClusterNotHealthy"
+    "annotations":
+      "message": "Cluster {{ $labels.cluster }} health status has been RED for at least 2m. Cluster does not accept writes, shards may be missing or master node hasn't been elected yet."
+      "summary": "Cluster health status is RED"
+    "expr": |
+      sum by (cluster) (opensearch_cluster_status == 2)
+    "for": "2m"
+    "labels":
+      "severity": "critical"
+  - "alert": "OpenSearchClusterNotHealthy"
+    "annotations":
+      "message": "Cluster {{ $labels.cluster }} health status has been YELLOW for at least 20m. Some shard replicas are not allocated."
+      "summary": "Cluster health status is YELLOW"
```

Comment:
Leaving this one here for a next PR - To reduce false alarms, we should differentiate here between the following:
- "yellow": one or more replicas missing (permanent until scale up)
- "yellow": shards are in the process of relocating to different nodes (temporary until the process finishes)